### PR TITLE
Dead links

### PR DESCRIPTION
**Pull request by thibaultduponchelle**
Hello 😃 

A bunch of dead links/lists removal

### CODE REVIEW

File: docs/lists/data/lists.json [hunk 1]
```diff
@@ -37,25 +37,6 @@
       "unsub" : "http://groups.google.com/group/ack-users",
       "url" : null
    },
-   "activeperl" : {
-      "archive" : "https://code.activestate.com/lists/activeperl/",
-      "category" : "Win32 Ports",
-      "comments" : "",
-      "date_last_confirmed_active" : "2015-01-27 12:45:00",
-      "defunct" : "0",
```
Comment (rspier) [inline docs/lists/data/lists.json:45]: instead of deleting the list, mark it as "defunct" for archival purposes.

**Review by rspier — CHANGES_REQUESTED** [review]
This is amazing.  

Please update the CL to mark the lists as defunct instead of deleting them, and then I'll merge it.  Thanks!

File: docs/lists/data/lists.json (hunk 1 above)
Comment (thibaultduponchelle) [reply docs/lists/data/lists.json:45]: Hello, thank you for the hint, I noticed that some lists were in the list but not in the perl.org nor my local dev website but haven't connected to this "variable".

**Comment by thibaultduponchelle** [comment]
I wanted to update this PR by pushing again commits on the source branch, but it's not working (probably because I deleted the fork after the initial PR the recreated it from the pull request) so I will close this PR and open another one beside, sorry for that.

**Pull request closed without merging**